### PR TITLE
Clarify publication rules

### PR DESCRIPTION
**Pull request by pantierra**
Lately I ran into some misunderstandings on the rules established for chaos.social. After some communication with @rixx, things became clearer to me. Here, I try to give back, with a reformulation of the same rules, but a bit more on details.

This would have helped me a lot to understand before signing up. I hope these changes help people in the future and this it may save time the admins explaining people the publication rules.

Thanks.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -22,13 +24,19 @@
 That includes behaviour from other instances - we have no problem with stopping to federate with asshole instances, or suspend contact with single accounts.
 </p>
 
-<hr>
+<h2>Publication rules</h2>
+
+We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance and the Fediverse in general a place of dialogue and interaction. Therefore we established some rules. Please be aware, that any repeated disregard of these rules will lead to silencing or suspension of your account.
```
Comment (ccoenen) [inline about.html]: perhaps it could be added that people will get some kind of notification beforehand. Right now it sounds like a single offense will get you silenced (without any mentioning of it). And I don't believe this is what happens.

Comment (rixx) [reply about.html]: No, you're right, that will not happen for things like crossposting. We always try about three times to reach people first. I'll try to find a way to make that clear.

File: about.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<h2>Code of Conduct</h2>
```
Comment (rixx) [inline about.html]: I think "General Behaviour" would be a better title here.

Comment (rixx) [reply about.html]: Also, this heading should be placed below the first paragraphs.

File: about.html (hunk 1 above)
Comment (rixx) [reply about.html]: The second sentence is superflous, and the third is, as well, as we already say in the very beginning that misbehaviour can lead to silencing or suspension of accounts.

File: about.html [hunk 3]
```diff
@@ -22,13 +24,19 @@
...
-<hr>
+<h2>Publication rules</h2>
+
+We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance and the Fediverse in general a place of dialogue and interaction. Therefore we established some rules. Please be aware, that any repeated disregard of these rules will lead to silencing or suspension of your account.
+
+<h3>Bots</h3>
+
+Configure all automated posts, like bots, to be <strong>unlisted</strong>. This means those posts can be seen by everybody but do not show up on any <a href="https://chaos.social/web/timelines/public/local">local</a> or <a href="https://chaos.social/web/timelines/public">federated timeline</a>.
+
+<h3>Third party content</h3>
```
Comment (rixx) [inline about.html]: I'd remove this sub-heading and replace "Bots" with "Bots and automated posts", since the issue is not with "third-party content", which is a bad term for what's going on here.

File: about.html (hunk 1 above)
Comment (pantierra) [reply about.html]: Yes, it is said for the Code of Conduct which measures will be taken. I was not expecting that the same enforcement also applies to the **Publication rules**.

Maybe we want to reformulate this a bit. Will give it a try.

File: about.html (hunk 3 above)
Comment (pantierra) [reply about.html]: I see this as a crucial difference: third party content, which gets added, is in my opinion 
something very different than bots. For example one of the main differences are the motivation why you don't want it on your platform.

* **Bots**: because they are automated and not humans.
* **Third-party content**: because you don't want half of your local timeline to show content coming from Twitter (athough they are humans). At least this was the rationale I was given by the admins.

This was the main confusion I had, when I read the sentence as it is currently. Because I was not expecting that content from Twitter would be seen as bad as bots (because it is content from humans), and the argumentation was completely mixed up and not understandable - at least for me.

File: about.html (hunk 2 above)
Comment (pantierra) [reply about.html]: I don't think that the **Publication rules** have anything to do with "being excellent to each other", especially not the part about third-party-content.

Comment (pantierra) [reply about.html]: Changed the title from "Code of Conduct" to "General behaviour" as suggested and placed the enforcement paragraph on top of the whole document, with a slight rephrasing to make clear it applies to all.

File: about.html (hunk 1 above)
Comment (pantierra) [reply about.html]: Reformulated so the "enforcement" paragraph is at the beginning and is understandably to be applied to all rules in **General behaviour** (Code of Conduct) and **Publication rules**.

**Comment by pantierra** [comment]
Thanks for your review @rixx and @ccoenen! I tried to incorporate your feedback.

File: about.html (hunk 1 above)
Comment (pantierra) [reply about.html]: Based on @ccoenen feedback I also added a sentence into the enforcement paragraph: *Usually, we'd tell you a couple of times in advance, before taking any of these measures.*

File: about.html (hunk 3 above)
Comment (pantierra) [reply about.html]: Sorry, I'm new to Mastodon - and I just discovered my deepest fault and confusion. @rixx explained to me, that also native boosts on Mastodon are generally handled as unlisted. So why should Twitter then be something better. Then, of course, it makes completely sense what you are saying @rixx, but for the people coming from Twitter I suggest to explain it briefly.

Comment (rixx) [reply about.html]: I don't quite agree – our about/more page is on the long side already. The longer the page, the less people will actually read it. Explaining how Mastodon works is not the job of the about page.

File: about.html [hunk 4]
```diff
@@ -1,7 +1,10 @@
+These are our house rules. Please read them before signing up. Any violation of any of those may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators. Usually, we'd tell you a couple of times in advance, before taking any of these measures.
```
Comment (rixx) [inline about.html]: It was a very conscious decision to start our about page with "Be excellent to each other". This paragraph should e moved up to be the introduction instead of this one.

File: about.html (hunk 3 above)
Comment (rixx) [reply about.html]: I'd also say that subheadings on h3 level make the page harder to read, and simply putting <strong> tags around the relevant words of a paragraph would work better.

Comment (pantierra) [reply about.html]: So, I gave it a try. Now it would be understandable for a newbie like me. What do you think?

File: about.html (hunk 4 above)
Comment (pantierra) [reply about.html]: Yes it makes sense. I also like to have this sentence first. I tried to rephrase it to have it first but still make the enforcement paragraph understandably applikable to all rules (including the automated posts).

File: about.html (hunk 3 above)
Comment (pantierra) [reply about.html]: Removed `<h3>` level headers.

File: about.html [hunk 5]
```diff
@@ -22,13 +25,15 @@
...
+
+<strong>Note on "Retweets":</strong>
 
-We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance a place of dialogue and interaction.
-Therefore we'd like to ask you to keep all <strong>automated posts unlisted</strong>. Unlisted posts can be seen by everybody but do not show up on the local timeline.
-This extends to bots, feed posters, Twitter "retweets", and Twitter crossposts with broken mentions.
+In Mastadon and the Fediverse "Boosts", which are similar to "Retweets" in Twitter, are handled a bit differently: They are always published as "unlisted", which means they won't appear neither on the <a href="https://chaos.social/web/timelines/public/local">local</a> nor on the <a href="https://chaos.social/web/timelines/public">federated timeline</a>. Therefore we don't tolerate "Retweets" that are not marked as "unlisted".
```
Comment (rixx) [inline about.html]: I still think that "This means they will be only visible on your personal timeline and for your followers.", and the complete "note on retweets" has no place in the /about. That would belong in a user guide for Mastodon (sp.), not in the rules section of an instance. As I said: the longer this section becomes, the fewer people will read it.

Comment (pantierra) [reply about.html]: Yes, you are right. It would be better placed in some "FAQ" or user guide. I just expect others to run into this frequently, because a lot of people will be coming from Twitter. And with you having at least a place to reference to that explains this shortly, the less you will loose time explaining it to newbies.

My goal is to help you as much as possible with this. And being a newbie writing "documentation" I consider a good way to support. Because small things like these are probably more than clear to you. But definitively not to people migrating from Twitter in such a wave as it happens currently. So if you want to take it out, I'm happy to do so. But I really think it would help people to get started much nicer than a harsh toot in the timeline.

Comment (pantierra) [reply about.html]: I also added an issue in the crossposter that I use to change the default behaviour (https://github.com/foozmeat/moa/issues/62). This will hopefully also have a relevant impact.

**Comment by rixx** [comment]
I pulled all rules into a single list, so that it won't be as easy to overlook it anymore, as per #9. Thank you anyways for your feedback and expressing what you were unclear on, this will help us communicate similar issues in the future!
